ledger-tool/src/main.rs

```diff
@@ -923,6 +926,25 @@ fn load_bank_forks(
         vec![non_primary_accounts_path]
     };
 
+    let mut accounts_update_notifier = Option::<AccountsUpdateNotifier>::default();
+    if arg_matches.is_present("geyser_plugin_config") {
+        let geyser_config_files = values_t_or_exit!(arg_matches, "geyser_plugin_config", String)
+            .into_iter()
+            .map(PathBuf::from)
+            .collect::<Vec<_>>();
+
+        let (confirmed_bank_sender, confirmed_bank_receiver) = unbounded();
+        drop(confirmed_bank_sender);
```

Comment:
@lijunwangs can you comment on whether it's ok to setup the geyser plugin service without slot/block notifications? It looks like we expect to send block metadata notifications whenever account data notifications are enabled: https://github.com/solana-labs/solana/blob/4608155588fef9d4c7ffd5aba819025bc8cbbe6c/geyser-plugin-manager/src/geyser_plugin_service.rs#L109

If it's ok, I think it would be better to change the `confirmed_bank_receiver` arg in `GeyserPluginService::new` to an `Option` and pass `None` instead of a receiver that will never be triggered.

Reply:
There is currently no means to disable block/slot notifications. The plugin implementation can itself just ignore the calls when called if they decided it is needed. I think at least the slot status is needed to make determination of the accounts data, otherwise we would not know if the accounts update are really accepted by the network.

Reply:
@terorie have you looked into setting up a slot status notification sender that can be used by the ledger tool? Or do you not need it for your use-case?

Reply:
> have you looked into setting up a slot status notification sender that can be used by the ledger tool?

@jstarry my end-goal is making the `ledger-tool verify` callback stream a drop-in replacement for `validator`. I thought it makes sense to split functionality into separate PRs but I'm happy to add all 3 (slot updates, accounts, transactions) to this one.

> Or do you not need it for your use-case?

I only need account updates (for https://bpf.wtf/sol-state-history/) but a few users on CT reached out saying they want to backfill their Geyser infra with ledger-tool. I can see the value in only having to build a single Geyser integration for both history & real-time indexing.

Reply:
Ok, I'm happy with separate PR's, just wanted to make sure this PR puts geyser support in a useful state, which it seems it does